src/channel.rs

```diff
@@ -368,7 +369,7 @@ impl ChannelBuilder {
         self
     }
 
-    /// Set a raw int configuration.
+    /// Set a raw Int configuration.
```

Comment:
What is `Int`?

Reply:
It should be integer? `Int` for abbr?

Reply:
Then why not use `integer`? And I believe its abbreviation is `int` instead of `Int`.

Reply:
Yes, it is integer now.